.env.sample

```diff
@@ -3,6 +3,9 @@
 # The absolute base url
 NEXT_PUBLIC_BASE_URL=http://localhost:3000
 
+# Supabase will redirect to this url after recovery
+NEXT_PUBLIC_RECOVERY_AUTH_REDIRECT_URL=http://localhost:3000/auth
```

Comment:
I've realized we have introduced a duplication here: the `http://localhost:3000` part is always the same as `NEXT_PUBLIC_BASE_URL`.

I would suggest we change this line to `NEXT_PUBLIC_RECOVERY_AUTH_REDIRECT_PATH=/auth` and then combine it with the `NEXT_PUBLIC_BASE_URL` in the code.

Reply:
Great idea, will fix 👍

Reply:
suggestion: how about we keep how it is in the code, and use this in the .env instead:
`NEXT_PUBLIC_RECOVERY_AUTH_REDIRECT_URL=${NEXT_PUBLIC_BASE_URL}auth`

what do you think?

Reply:
I'm not sure if this works in our production environment. I have a feeling that Vercel doesn't allow for one variable to reference another. But feel free to check it out!

Reply:
Next.js can do variable expansion:

https://nextjs.org/docs/basic-features/environment-variables#loading-environment-variables

>Note: Next.js will automatically expand variables ($VAR) inside of your .env* files. This allows you to reference other secrets, like so:

```plain
# .env
HOSTNAME=localhost
PORT=8080
HOST=http://$HOSTNAME:$PORT

```

Reply:
Try the notation ${VAR_NAME}. It should be valid, at least it is on posix shells.